core/app/components/forms/FormLinkCopyButton.tsx

```diff
@@ -0,0 +1,10 @@
+"use client";
+
+import type { CopyButtonProps } from "ui/copy-button";
+import { CopyButton } from "ui/copy-button";
+
+type Props = Omit<CopyButtonProps, "value">;
+
+export const FormLinkCopyButton = (props: Props) => {
```

Comment:
nit: what do you think about making this a general `CopyCurrentLinkButton` ? since there's nothing in here specific to form linking, so then we can easily reuse it elsewhere too

Reply:
Great suggestion! I'll do that now.